2022/21xxx/CVE-2022-21504.json

```diff
@@ -1,17 +1,76 @@
+
```

Comment:
('Expecting property name enclosed in double quotes: line 34 column 25 (char 1180)',)